content/03-ventures/01-purpose.md

```diff
@@ -0,0 +1,13 @@
+# Purpose
+
+## What is a Venture?
```

Comment:
I'd use the Chicago Manual of Style for the capitalization of titles. See [title.sh](https://title.sh) for an easy tool to generate them.

```suggestion
## What Is a Venture?
```

Reply:
Sounds useful to take this as a guideline. Is there are a good resource to scan through the Chicago Manual?

Reply:
You can read more about it here: http://www.chicagomanualofstyle.org/